src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -2996,66 +2996,28 @@ private StatementAst ConfigurationStatementRule(IEnumerable<AttributeAst> custom
                         {
                             // Load the default CIM keywords
                             Collection<Exception> CIMKeywordErrors = new Collection<Exception>();
-                            if (ExperimentalFeature.IsEnabled("PSDscJsonSchemaSupport"))
+                            if (ExperimentalFeature.IsEnabled(Microsoft.PowerShell.DesiredStateConfiguration.Internal.Json.DscClassCache.DscExperimentalFeatureName))
```

Comment:
Consider a `using` statement

Reply:
Replaced all instances in the file with a namespace alias:
`using Dsc = Microsoft.PowerShell.DesiredStateConfiguration.Internal;`